app/src/main/res/values/settings_keys.xml

```diff
@@ -28,6 +28,25 @@
     <string name="screen_brightness_key" translatable="false">screen_brightness_key</string>
     <string name="screen_brightness_timestamp_key" translatable="false">screen_brightness_timestamp_key</string>
 
+    <string name="seek_duration_key" translatable="false">seek_duration</string>
+    <string name="seek_duration_default_key" translatable="false">10000</string>
```

Comment:
This variable should probably be called `seek_duration_default_value`. You should be good by just committing this and the below suggestion directly.
```suggestion
    <string name="seek_duration_default_value" translatable="false">10000</string>
```

Reply:
Correct. Good catch :+1: